implementations/rust/ockam/ockam_transport_tcp/tests/send_receive.rs

```diff
@@ -86,10 +86,12 @@ async fn tcp_lifecycle__reconnect__should_not_error(ctx: &mut Context) -> Result
     // TcpSender address should not exist
     let res = child_ctx.send(tx_address.clone(), "TEST".to_string()).await;
     assert!(res.is_err());
-    assert_eq!(
-        res.err().unwrap(),
-        ockam_node::error::Error::UnknownAddress.into()
-    );
+
+    // FIXME!
+    // assert_eq!(
```

Comment:
This is the point of error codes, actually, and once downcasting is supported (I describe how it will be done in the issue), you can get the exact error value back.